packages/plugin-interactive-tools/tests/__snapshots__/colorizeUtils.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`colorizeUtils colorizeVersionDiff should handle semver with prerelease part 1`] = `"^2.0.0[35m-beta.18[39m"`;
```

Comment:
It is hard to test the output after prettier with normal `expect` assert. so I resolve to snapshot.